sdk/cosmos/azure-cosmos-spark_3_2-12/Samples/Scala/NYC-Taxi-Data/01_Batch.scala

```diff
@@ -255,7 +255,7 @@ val readCfg = Map(
   "spark.cosmos.accountKey" -> cosmosMasterKey,
   "spark.cosmos.database" -> "SampleDatabase",
   "spark.cosmos.container" -> "GreenTaxiRecords",
-  "spark.cosmos.read.partitioning.strategy" -> "Default",
+  "spark.cosmos.read.partitioning.strategy" -> "Restrictive",
```

Comment:
will this cause regressions for customers who is using 'Default' for the high concurrency?

Reply:
Technically possible - but extremely unlikely given the high RU charges.